src/PlanktonIndividualExample.jl

```diff
@@ -0,0 +1,1280 @@
+md"""
+We run the model for 120 time steps (2 hours) and then plot individuals and nutrients in their final state (stored in model).
+"""
+
+# ╔═╡ c79a07b2-e448-44d8-b0a0-c42b5fc1db89
+for i in 1:120
+    update!(sim)
+end
+
+# ╔═╡ ce973a4d-b612-450a-8dd9-ddcfbec59c65
+md"""
+To plot the distribution of individuals as well as nutrient fields we use Plots.jl and create a function that can easily be re-used e.g. to create an animation.
+"""
+
+# ╔═╡ d87581c9-63ec-4519-8c84-f8442f05e0a8
+function plot_model(model::PlanktonModel)
+    ## Coordinate arrays for plotting
+    xC, zC = collect(model.grid.xC)[3:130], collect(model.grid.zC)[3:130]
+
+    ## contour of the flow field
+    fl_plot = Plots.contourf(xC, reverse(zC), rotl90(ϕcenters), xlabel="x (m)", ylabel="z (m)", color=:balance, fmt=:png, colorbar=false)
+
+    ## a scatter plot embeded in the flow fields
+    px = Array(model.individuals.phytos.sp1.data.x) .* 1 # convert fractional indices to degree
+    pz = Array(model.individuals.phytos.sp1.data.z) .* -1# convert fractional indices to degree
+    Plots.scatter!(fl_plot, px, pz, ms=5, color = :red, legend=:none)
+
+    ## DOC field
+    trac1 = Plots.contourf(xC, reverse(zC), rotl90(Array(model.nutrients.DOC.data)[3:130,3,3:130]), xlabel="x (m)", ylabel="z (m)", clims=(0.5, 1.1), fmt=:png)
+
+    ## Arrange the plots side-by-side.
+    plt = Plots.plot(fl_plot, trac1, size=(800, 400),
+        title=[lpad(model.t÷86400,2,"0")*"day "*lpad(model.t÷3600-24*(model.t÷86400),2,"0")*"hour" "DOC (mmolC/L)"])
+
+    return plt
+end
+
+# ╔═╡ 581809a1-6f8b-40d0-9a6e-d943c424be07
+plot_model(model)
+
+# ╔═╡ 3073ceb2-dccb-400e-ae0d-8ff73569cd54
+md"""
+You can also generate a animation if you like.
+"""
+
+# ╔═╡ 2c119e21-afbf-48ae-aebe-fc161c18f344
+anim = @animate for i in 1:120
+	update!(sim)
+   plot_model(model)
+end
+
+# ╔═╡ 63b46d92-d915-492a-807b-af99e6e298c0
+gif(anim, "anim_fps15.gif", fps = 15)
+
+# ╔═╡ a685d6e7-88aa-427b-be61-f1e5da15653a
+gif(anim, "anim_fps15.gif", fps = 15)
+
+# ╔═╡ fa502a3f-ced0-4e82-afce-3baf4cacf4c7
+md"""
+## 5. Exercise
+"""
+
+# ╔═╡ af8be494-afdf-4e2e-a1be-b069c3a14f18
+md"""
+1. Add the second species
+2. Run for a longer period
+3. Add diagnostics
+4. ...
+"""
+
+# ╔═╡ 964cb129-9d7c-47bc-945a-7639e80da21f
+md"""
+#### 1. Add the second species
+
+Run the code below with `N_species = 2` to add the second species.
+
+```julia
+model = PlanktonModel(arch, grid; N_species = 2, N_individual = 2^7, max_individuals = 2^7*8)
+```
+
+In this model setup, there're **two** species, each species has `2^7` individuals.
+"""
+
+# ╔═╡ ed092d52-8afb-4c97-a1fe-4e6cbae33420
+md"""
+#### 2. Run for a longer period
+
+In order to run a longer period, you need first to change `ΔT_vel` to tell the model to use the flow fields for more time steps.
+
+For example, if we want to run the model for 1 day:
+
+```julia
+sim = PlanktonSimulation(model, ΔT = 60, nΔT = 60*24, vels=(u=uvels, v=vvels, w=wvels), ΔT_vel=60*60*24)
```

Comment:
Maybe it's just me, but I find the timesteps part confusing. This might be a stupid idea too so don't hesitate to dismiss it. What about using SciML lingo here instead? As in, use `tspan` and such?

I thought `ΔT_vel` was supposed to always be the same as `ΔT * nΔT`... But then if you set `nΔT == 1`, is it actually replaced under the hood by `nΔT = ΔT_vel ÷ ΔT`? (As you can tell I'm confused 🤔)

Reply:
Yes, `nΔT` might be confusing. We can use `stop_time` or `tspan`  instead for the naming.

And the reason we use `ΔT_vel` is that the time step of flow fields can be different from the time step of our model, that is to say, one time step of flow fields can cover multiple time steps of `PlanktonModel`.  So `ΔT_vel` can be the same as `ΔT` or some folds of `ΔT`.